include/slang.h

```diff
@@ -884,6 +884,7 @@ typedef uint32_t SlangSizeT;
         DumpWarningDiagnostics,
         InputFilesRemain,
         EmitIr,                        // bool
+        EmitReflectionJSON,            // bool
```

Comment:
This will break binary compatibility. New enum values should be added at the end.